### DISCUSSION TITLE
What's the prefered method to load/draw a blender scene for now?

### DISCUSSION THREAD

**Discussion by royqh1979** [raysan5/raylib#2300] [Q&A] [answered]
I noticed that raylib doesn't support load and draw scenes in glTF 2.0 files. So what's the prefered method to load/draw scenes exported from blender for now?

Or are there any tutorials about how to export  blender models that can be correctly loaded by raylib?

**Comment by raysan5** [comment, marked as answer]
@royqh1979 Actually there is no a preferred method because it looks like more of an engine feature than a low level library like raylib. 
The PR solution you proposed is really good but too high-level for raylib. 
As many other high-level features (https://github.com/JeffM2501/raylibExtras) it would be nice to have a higher-level library to do it, instead of being a part of the raylib core features.

**Reply by royqh1979** [reply]
Thanks. Any chance to add an array to struct Model , for each mesh's transform? In that way we can load a glTF model and use most of the transformations too.

**Reply by royqh1979** [reply]
Load scene may not be a feature that raylib should support. But I think load  mesh transformation infos in the nodes shoulde be supported by raylib. Because without mesh transformation  infos, the model loaded is not correct.
